### DISCUSSION TITLE
is there a demo showing how this works?

### DISCUSSION THREAD

**Discussion by hannesdelbeke** [coddingtonbear/obsidian-web#77] [General]
Hi, this extension sounds very promising.
Is there a demo or documentation page showing how this works?
The readme doesn't seem to give me a clear understanding of how this works.

**Comment by hannesdelbeke** [comment]
Thanks for moving this to the correct location!

**Comment by coddingtonbear** [comment]
This interacts with Obsidian over the REST API provided by `obsidian-local-rest-api`.  There is a live demo of how `obsidian-local-rest-api` works here: https://coddingtonbear.github.io/obsidian-local-rest-api/; this plugin is just a little UI overlaid atop that.
